README.md

```diff
@@ -76,7 +76,7 @@ function greet(input) {
 }
 ```
 
-Try it out at our [REPL](https://babel.dev/repl#?browsers=defaults%2C%20not%20ie%2011&loose=true&code_lz=GYVwdgxgLglg9mABAcwE4FN1QBQzABxCgEpEBvAKEUQyhFST0KkQH5XEAiACXQBs-cRAHc4qPgBNOAbgoBfIA&shippedProposals=true&sourceType=script&lineWrap=true&presets=env%2Cenv&prettier=true).
+Try it out at our [REPL](https://babel.dev/repl#?browsers=defaults%2C%20ie%2011&loose=true&code_lz=GYVwdgxgLglg9mABAcwE4FN1QBQzABxCgEpEBvAKEUQyhFST0KkQH5XEAiACXQBs-cRAHc4qPgBNOAbgoBfIA&shippedProposals=true&sourceType=script&lineWrap=true&presets=env%2Cenv&prettier=true).
```

Comment:
How about use `targets: defaults` and enable `Force All Transforms`?

Reply:
I'm not sure about this, maybe see what others think.

Reply:
In Babel 8 the `defaults` will the `targets` so chances are we are not transforming anything (the ultimate goal of preset-env) if all browsers support new features quickly.

I agree with @fisker. And we can also add examples using popular proposals e.g. decorators / pipelines.

Reply:
Oops, it occurred to me while doing this that this might expose some bugs.
For example like this
https://github.com/babel/babel/issues/14960#issuecomment-1253412072

In addition, the examples on the official website are also invalid. Maybe we can choose a new example, I personally prefer a short one because the official website is showing it dynamically.